### DISCUSSION TITLE
Feature Request: Sort order

### DISCUSSION THREAD

**Discussion by demlak** [go-shiori/shiori#332] [Feature requests]
Would be nice to sort the order of search and/or tag results. for example by date added.

**Comment by cocoastorm** [comment]
Hey, I would love to start working on this. But I'm not sure what functionality you would like for the sort order.

Checking out the API method [here](https://github.com/go-shiori/shiori/blob/master/internal/webserver/handler-api.go#L159)

Looks like there's options to change the OrderMethod from "Last Added" & "Last Modified"?

I was thinking of adding a `<select>` in the UI to toggle between these two sort options. What do you think?

Cheers!

**Comment by Code-Slave** [comment]
I think that would be a good start, maybe sort by tag alphabetically too
